runtime/gateway.go

```diff
@@ -440,20 +440,20 @@ func (gateway *Gateway) setupTChannel(config *StaticConfig) error {
 	serviceName := config.MustGetString("tchannel.serviceName")
 	processName := config.MustGetString("tchannel.processName")
 
+	subScope := gateway.MetricScope.SubScope("tchannel")
```

Comment:
Verified that tchannel metrics doesn't come with the "tchannel" prefix, also added integration tests.